src/gui/filedetails/FileDetailsWindow.qml

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2022 by Claudio Cambra <claudio.cambra@nextcloud.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+import QtQuick 2.15
+import QtQuick.Window 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+
+import com.nextcloud.desktopclient 1.0
+import Style 1.0
+
+ApplicationWindow {
+    id: root
+
+    property var accountState
+    property string localPath: ""
+
+    flags: Qt.Dialog
+    width: 400
+    height: 500
```

Comment:
you may also want to put a minimum window size to prevent it being squeezed too little and look bad

Reply:
True, I added a minimum width and height